SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialViewWorkerConnection.cpp

```diff
@@ -62,31 +64,35 @@ Worker_RequestId USpatialViewWorkerConnection::SendCreateEntityRequest(TArray<FW
 	{
 		Data.Emplace(SpatialGDK::OwningComponentDataPtr(Component.schema_type), Component.component_id);
 	}
-	return Coordinator->SendCreateEntityRequest(MoveTemp(Data), Id);
+	return Coordinator->SendCreateEntityRequest(MoveTemp(Data), Id, {}, SpanId);
 }
 
-Worker_RequestId USpatialViewWorkerConnection::SendDeleteEntityRequest(Worker_EntityId EntityId)
+Worker_RequestId USpatialViewWorkerConnection::SendDeleteEntityRequest(Worker_EntityId EntityId,
+																	   const TOptional<worker::c::Trace_SpanId>& SpanId)
 {
 	check(Coordinator.IsValid());
 	return Coordinator->SendDeleteEntityRequest(EntityId);
```

Comment:
Is it intentional that we don't pass `SpanId` here?

Reply:
Yes I think this was missed. Nice spot.